tests/e2e/helpers.ts

```diff
@@ -39,6 +39,41 @@ export const WALLETS = {
   },
 };
 
+/** Wallets created for transaction testing. */
+export const WALLETS_WITH_COINS = {
+  '3-Keys Vault': {
+    SEED_PHRASE: 'viable reason stable tongue sibling jazz virus dress despair tunnel truly heart',
+    FAST_KEY:
+      '0442d7724d90fb60bc969f8b0fd46f3f63fe17637d5a0ba2fa9800b3b85946b72c3b81199572cd91bad23c87c3e96dbaa68e1c4b3e47d09276bd63138c584a5a7b',
+    CANCEL_KEY:
+      '04e8bc5e2428dcebe434306adaa944cb5eb7df80ec2e544f94ab2cea9bc5a70b5b1af42a83a936cd9d277413a8c5303001beaa268724270e4f2ce4d62010421960',
+  },
+
+  '2-Keys Vault': {
+    SEED_PHRASE: 'judge void hair leader legal fantasy strong clinic knee child great situate',
+    CANCEL_KEY:
+      '0442d7724d90fb60bc969f8b0fd46f3f63fe17637d5a0ba2fa9800b3b85946b72c3b81199572cd91bad23c87c3e96dbaa68e1c4b3e47d09276bd63138c584a5a7b',
+  },
+
+  'Standard HD P2SH': {
+    SEED_PHRASE: 'roof jump holiday jaguar calm mammal vapor acoustic marine modify hope enemy',
+  },
+
+  'Standard P2SH': {
+    SEED_PHRASE: 'cNVqz3DVCUXUSbFWVuth7CUqmgaqCz7yck2kDjt5zgUxgLkK4WZi',
+  },
+
+  'Standard HD Segwit': {
+    SEED_PHRASE: 'patrol correct rural firm special eight smooth orphan pilot oppose wealth rival',
+  },
+};
+
+export const DATA_FOR_TRANSACTIONS = {
+  AMOUNT_TO_SEND: '0.0001',
+  WALLET_ADDRESS: '2MuY5McUz14TH6B3H9EypErpnFjNbNeqDVz',
+  CONFIRM_TRANSACTION_PASSWORD: 'qwertyui',
+};
```

Comment:
Few things:
- I'm wondering if we should extract `AMOUNT_TO_SEND` to a variable. It very depends on the test that we're making
- `WALLET_ADDRESS`, as the name suggests, should be a part of the wallet and I don't see a point in keeping it separate from other properties like `SEED_PHRASE`, `FAST_KEY`, and so on.
- `CONFIRM_TRANSACTION_PASSWORD` should just a variable, not a part of `DATA_FOR_TRANSACTIONS` object I believe. And the suffix `CONFIRM` isn't necessary - it's just a transaction password. The word `DEFAULT` match better here.

Reply:
Done